.gitlab/e2e/e2e.yml

```diff
@@ -254,7 +283,9 @@ new-e2e-updater:
   extends: .new_e2e_template
   rules:
     !reference [.on_updater_or_e2e_changes_or_manual]
-  needs: ["deploy_deb_testing-u7_arm64"]
+  needs:
+    - job: deploy_deb_testing-u7_arm64
+      optional: true
```

Comment:
Is there a specific reason to change it to `optional: true`?

Reply:
No it was an error